project/plugins.sbt

```diff
@@ -1,29 +1,35 @@
-resolvers += Classpaths.typesafeReleases
+resolvers += Resolver.typesafeRepo("releases")
 
 resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.11.2")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.3")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
 
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.3.2")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
 
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.7.0")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
 
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
 
 // publishing
 
 resolvers += "Era7 maven releases" at "http://releases.era7.com.s3.amazonaws.com"
 
-addSbtPlugin("ohnosequences" % "sbt-s3-resolver" % "0.10.1")
+addSbtPlugin("ohnosequences" % "sbt-s3-resolver" % "0.14.0")
 
 resolvers += Classpaths.sbtPluginReleases
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.0.4")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
-addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
+addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.10")
+
+resolvers += Resolver.sonatypeRepo("snapshots")
+
+libraryDependencies += "org.scalariform" %% "scalariform" % "0.2.0-SNAPSHOT"
```

Comment:
That one slipped through the cracks. There's a 0.2.0 coming soon with some
good formatting fixes (including correct double indent classes). Will
remove.

Still unsure why -some- nodes kill sbt when it's in integration test mode.
On Fri, Jun 24, 2016 at 3:10 PM Matthias Veit <notifications@github.com>
wrote:

> In project/plugins.sbt
> <https://github.com/mesosphere/marathon/pull/4025#discussion_r68395275>:
>
> >
> > -addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
> > +addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
> > +
> > +addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.10")
> > +
> > +resolvers += Resolver.sonatypeRepo("snapshots")
> > +
> > +libraryDependencies += "org.scalariform" %% "scalariform" % "0.2.0-SNAPSHOT"
>
> why do we need this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4025/files/bfa987cdc4877151aa832225336e77a4e9798ac4#r68395275>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAweZnz0mP1uKwBRDYOMxWVmuWrRCsnUks5qO9dPgaJpZM4I9lue>
> .
>